src/main/java/org/opentripplanner/middleware/otp/OtpDispatcherResponse.java

```diff
@@ -90,4 +99,34 @@ public OtpDispatcherResponse clone() {
 //        clonedObject.responseBody = this.responseBody;
 //        return clonedObject;
     }
+
```

Comment:
Not strictly part of this PR, but what is happening with the clone method above this? I think the comments should at least be removed.

Reply:
I don't know... Those changes are from commit 38dafa4b71983ae5f2b8f47bc6e7568007ba5fb1 by @landonreed.